src/include/planner/plannodes/nested_loop_join_plan_node.h

```diff
@@ -69,6 +114,17 @@ class NestedLoopJoinPlanNode : public AbstractJoinPlanNode {
 
   nlohmann::json ToJson() const override;
   std::vector<std::unique_ptr<parser::AbstractExpression>> FromJson(const nlohmann::json &j) override;
+
+ private:
+  /**
+   * Left keys
```

Comment:
nit: Expand more, what are the left keys used for, what makes them different from right keys. Do the same for `right_keys_`